### PR TITLE
Chapter 1/Homework 3 

### PR DESCRIPTION
**Pull request by Poordevv**
I was a little confused at first but eventually, I figured I had to change my previous homework 1 & 2 source code to match up with Checks.cs. Anyway, if your reading this, have a great day!

### CODE REVIEW

File: Src/BootCamp.Chapter/Checks.cs [hunk 1]
```diff
@@ -16,26 +16,27 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            // To do: call your implementation.
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Checks.cs:19]: Comments can be removed after implementation.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                var name = promptStringValue("Name: ");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:13]: Loop body can be replaced with a function to ProcessPersonBmi or similar

**Comment by Almantask** [comment]
> I was a little confused at first but eventually, I figured I had to change my previous homework 1 & 2 source code to match up with Checks.cs. Anyway, if your reading this, have a great day!

Cheers! 😄

File: Src/BootCamp.Chapter/Checks.cs (hunk 1 above)
Comment (Poordevv) [reply Src/BootCamp.Chapter/Checks.cs]: Removed.

**Comment by Almantask** [comment]
Looks good 😄

**Pull request closed without merging**